### PR TITLE
Feat: Added sample api for test flow [#1642]

### PR DESCRIPTION
**Pull request by Abhishek9503**
## Description

This PR added the tour guide sample api portion in which a sample api is added to the tour guide when user runs the tour guide .  #1642 
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

File: apps/@sparrow-desktop/src/pages/workspace-page/sub-pages/TestflowExplorerPage/TestflowExplorerPage.ViewModel.ts [hunk 1]
```diff
@@ -299,6 +303,43 @@ export class TestflowExplorerPageViewModel {
...
+  public handleTestFlowRunEnd = () => {
+    const progressiveTab = createDeepCopy(this._tab.getValue());
+    testFlowDataStore.update((testFlowDataMap) => {
+      // let wsData = testFlowDataMap.get(progressiveTab.tabId);
+      testFlowDataMap.delete(progressiveTab.tabId);
+      // testFlowDataMap.set(progressiveTab.tabId, wsData);
+      return testFlowDataMap;
+    });
+  };
+
```
Comment (itsmdasifraza) [inline apps/@sparrow-desktop/src/pages/workspace-page/sub-pages/TestflowExplorerPage/TestflowExplorerPage.ViewModel.ts:342]: not in use

File: packages/@sparrow-workspaces/src/components/table-navbar/TableNavbar.svelte [hunk 2]
```diff
@@ -4,10 +4,18 @@
...
 
   export let selectedNode;
+
   export let onClose;
   export let onRedirect;
+
+  export let handleTestFlowRunEnd;
+
+  onMount(() => {
+    console.log("THis is selected NOde ", selectedNode);
```
Comment (itsmdasifraza) [inline packages/@sparrow-workspaces/src/components/table-navbar/TableNavbar.svelte:17]: console

File: packages/@sparrow-workspaces/src/features/testflow-explorer/components/select-api/SelectAPIRequest.svelte [hunk 3]
```diff
@@ -72,48 +76,100 @@
...
+      ],
+    },
+  ];
+
+  let showSampleApi = false;
+
+  const test = () => {
+    arrayData = collections;
+  };
+
```
Comment (itsmdasifraza) [inline packages/@sparrow-workspaces/src/features/testflow-explorer/components/select-api/SelectAPIRequest.svelte:143]: improve function name

File: packages/@sparrow-workspaces/src/components/table-navbar/TableNavbar.svelte (hunk 2 above)
Comment (Abhishek9503) [reply packages/@sparrow-workspaces/src/components/table-navbar/TableNavbar.svelte:17]: Resolved

File: packages/@sparrow-workspaces/src/features/testflow-explorer/components/select-api/SelectAPIRequest.svelte (hunk 3 above)
Comment (Abhishek9503) [reply packages/@sparrow-workspaces/src/features/testflow-explorer/components/select-api/SelectAPIRequest.svelte:143]: Resolved

File: apps/@sparrow-desktop/src/pages/workspace-page/sub-pages/TestflowExplorerPage/TestflowExplorerPage.ViewModel.ts (hunk 1 above)
Comment (Abhishek9503) [reply apps/@sparrow-desktop/src/pages/workspace-page/sub-pages/TestflowExplorerPage/TestflowExplorerPage.ViewModel.ts:342]: Removed

**Review by Astitva877 — APPROVED** [review]

**Pull request merged**